packages/dev_disc/README.md

```diff
@@ -64,9 +64,9 @@ devsim_cmd_msg_pipe = /var/tmp/liota/devsim_cmd_messenger.fifo	# the named pipe
 
 							# if no item in this list, Device Discovery will not be started
 
-disc_msg_pipe = /var/tmp/liota/discovery_messenger.fifo		# currently, only support these 4 types
+disc_msg_pipe = /var/tmp/liota/discovery_messenger.fifo		# currently, support these 4 types (currently,
 
-socket = 127.0.0.1:5000						# you can only use some of them by deleting others
+socket = 127.0.0.1:5000						# coap and socket are not allowed for security consideration).
 
 mqtt = 127.0.0.1:1883:device_discovery				# IP address should be updated according to your system
```

Comment:
If TLS/cert check is not used on loopback, the port number should be below 1024. Otherwise another process can connect to 1883, pretend to be MQTT endpoint and steal the credentials passed by clients.